spec/amp-actions-and-events.md

```diff
@@ -454,7 +459,7 @@ event.response</pre></td>
   </tr>
 </table>
 
-### amp-lightbox
+### amp-lightbox <a name="amp-lightbox-1"></a>
```

Comment:
Curious, why do some of these names end with `-1` and others don't?

Reply:
looks like the `-1` is added to ones that have info under both actions and events.